CHANGELOG.md

```diff
@@ -0,0 +1,133 @@
+# changelog
+
+## unreleased
+
+### added
+ - Added edge apodization for LSST.
+
+### changed
+ - LSST codes internally use DM exposures as opposed to ngmix observations.
+ - All PSF fits are done only with adaptive moments.
+
+### removed
+ - Removed unused deblender code for LSST.
+
+### fixed
+ - Fixed bug in LSST jacobians.
+
+
+## 0.4.0 - 2021-11-17
+
+### added
+ - Weight masks now get 4-fold symmetry applied to zero-weight pixels before calling moments fitters.
+ - Routines to do metacalibration directly with LSST exposure objects and galsim
+ - Image edges are now apodized to reduce FFT artifacts for bright sources on the edges of images.
+ - LSST codes now apodize circular masks applied to bright objects.
+ - LSST codes now have a simple EM deblender for testing.
+ - LSST codes now support multiple bands.
+
+### changed
+ - Shear values outside of |g| < 1 or |g1| < 1 or |g2| < 1 are now flagged in the fitters
+ - The maskflags parameter was renamed to nodet_flags
+
+### fixed
+ - Scarlet's internal cache is now cleared to prevent memory usage issues
+
+
+## 0.3.11 - 2021-10-20
+
+### added
+ - Added working implementations of LSST versions with scarlet.
+
+### changed
+ - Internal refactoring of moments measures. Now supports pgauss.
+
+### removed
+ - Dropped support for Gaussian fits.
+ - Dropped support for ngmix 1.0
+
+
+## 0.3.10 - 2021-10-02
+
+### changed
+ - Refactored LSST code to a functional approach.
```

Comment:
Note the code isn't "functional" in a technical sense, it is what experts would call imperative/procedural.

https://en.wikipedia.org/wiki/Functional_programming
https://en.wikipedia.org/wiki/Imperative_programming

Reply:
I never follow all of these things. :(